yt/config.py

```diff
@@ -92,15 +92,12 @@ def callback(leaf):
         return self.config_root.get_leaf(section, *keys, callback=callback)
 
     def get_most_specific(self, section, *keys, **kwargs):
-        use_fallback = "fallback" in kwargs
-        fallback = kwargs.pop("fallback", None)
         try:
             return self.config_root.get_deepest_leaf(section, *keys)
         except KeyError as err:
-            if use_fallback:
-                return fallback
-            else:
-                raise err
+            if "fallback" in kwargs:
+                return kwargs["fallback"]
+            raise err
```

Comment:
this is not _exactly_ replicating the existing behaviour because the `kwargs` dict isn't modified in place as with `dict.pop`. I'm confident that it's not important, but just for safety I'd like @cphyc 's review specifically here :)

Reply:
It's much better not to alter `kwargs` at all (as you're doing here) rather than pop'ing arguments (as I was doing).

Reply:
Cool, thanks !